nautobot/ipam/filters.py

```diff
@@ -404,7 +400,7 @@ class IPAddressFilterSet(
 
     class Meta:
         model = IPAddress
-        fields = ["id", "ip_version", "dns_name", "tags"]
+        fields = ["id", "ip_version", "dns_name", "tags", "mask_length"]
```

Comment:
This makes `mask_length` into a proper MultiValueNumberFilter right? Should that be documented in the migration tables?